packages/core/src/indexing/service.ts

```diff
@@ -1,4 +1,4 @@
-import type { IndexingFunctions } from "@/build/functions/functions.js";
+import type { IndexingFunctions } from "@/build/configAndIndexingFunctions/configAndIndexingFunctions.js";
```

Comment:
This is an unfortunately long file name - I wonder if perhaps we don't need the additional `schema` and `configAndIndexingFunctions` folders within the `build` dir now.